lms/static/sass/views/_teams.scss

```diff
@@ -0,0 +1,259 @@
+.view-teams {
+
+  // resets for existing styles
+  div.course-wrapper section.course-content {
+    padding: 0;
+
+    p {
+      margin-bottom: ($baseline/4);
+    }
+  }
+
+  .icon {
+    font-family: "FontAwesome";
+  }
+
+// layout
+  .page-header {
+    @include clearfix();
+    padding: $baseline $baseline ($baseline/2) $baseline;
+    border-bottom: 3px solid $m-blue-l1;
+  }
+
+  .header-main {
+    display: block;
```

Comment:
This is a ``<div>`` right? Does it need this or does the user agent/browser styling already get you this?